spec/openstudios_subdomain_helper.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+before do
```

Comment:
does this not mess up all the other tests?  also should/could it go under `spec/support`?

we might want a meta tag to configure tests that should be in the subdomain.  You can see the way we setup elastic search dependent tests in `support/elasticsearch`

Once the meta tag rules are setup in the support file, we can do something like
```
describe ArtistMailer, elasticsearch: :stub do
```

so I could imagine
```
describe SubdomainDependentThing, subdomain: true
```
or
```
describe SubdomainDependentThing, subdomain: 'openstudios'
```

Reply:
I like this a whole lot better 👍

Reply:
This is a noop